05/app.js

```diff
@@ -0,0 +1,71 @@
+const student = new Student('Jan', 'Kowalski')
+
+function Student(initFirstName, initLastName) {
+  this.firstName = initFirstName;
+  this.lastName = initLastName;
+  this.grades = {}
+}
+
+student.addGrade = function(subject, grade) {
+  if(typeof this.grades[subject] === 'undefined') {
+    this.grades[subject] = [];
+}
+   this.grades[subject].push(grade)
+}
+
+student.addGrade("math", 5)
+
+student.addGrade("math", 5)
+
+student.addGrade("polski", 20)
+
+student.addGrade("polski", 40)
+
+ 
+
+student.getAverageGrade = function(subject) {
+  let sum = 0;
+  if (typeof subject != "undefined") {
+    const grade = this.grades[subject]
```

Comment:
Skoro tutaj przypisałeś tą wartość to poniżej możesz wykorzystać.

PS. Osobiście użyłbym liczby mnogiej bo to kilka ocen.